lib/ui/index.ts

```diff
@@ -192,3 +192,7 @@ export function uiDeprecatedMessage(
   uiDeprecatedTag(tag);
   logger.log();
 }
+
+export function indent(str: string, spacing?: string) {
```

Comment:
There's actually already an `indent` function in the `Diagnosis` class used in doctor - could probably pull that one out into `lib/ui`. I think the way it accepts a number of indentions rather than a string might be a bit easier to use as well